estate/models/estate_property.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate property descripction"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default = fields.Date.add(fields.Date.today(), months=+3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facedes = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')]
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        string='State',
+        selection = [('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accpeted'), ('sold', 'Sold'), ('cancelled', 'Cancelled')],
+        default = 'new'
+    )
```

Comment:
```suggestion
    state = fields.Selection([
        ("new", "New"), 
        ("offer_received", "Offer Received"), 
        ("offer_accepted", "Offer Accepted"), 
        ("sold", "Sold"), 
        ("canceled", "Canceled")
    ], string="State", required=True, copy=False, default="new")
    )
```
just a matter of coding style. to gain some lines without being unreadable :)
also, don't use white spaces for the technical name of your selection options.